### PR TITLE
[ISSUE-724] kafka-ssl-sink kamelet

### PR DESCRIPTION
**Pull request by agiertli**

### CODE REVIEW

**Review by claudio4j — CHANGES_REQUESTED** [review]

File: kamelets/kafka-ssl-sink.kamelet.yaml [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+
+      The Kamelet is able to understand the following headers to be set:
+
+      - `key` / `ce-key`: as message key
+
+      - `partition-key` / `ce-partitionkey`: as message partition key
+
+      Both the headers are optional.
+    properties:
+      bootstrapServers:
```
[the remark this answers is not in the document]
Comment (oscerd) [reply kamelets/kafka-ssl-sink.kamelet.yaml:42]: Yes, +1. Just for maintain the same "style"

File: kamelets/kafka-ssl-sink.kamelet.yaml [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+        title: SSL Keystore Password
+        type: string
+      sslTruststoreLocation:
+        description: The location of the trust store file.
+        title: SSL Truststore Location
+        type: string
+      sslKeyPassword:
+        description: The password of the private key in the key store file.
+        title: SSL Key Password
+        type: string
```
[the remark this answers is not in the document]
Comment (oscerd) [reply kamelets/kafka-ssl-sink.kamelet.yaml:87]: This is important for tooling side.

File: kamelets/kafka-ssl-sink.kamelet.yaml [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+                      simple: ${header[partition-key]}
+              - simple: ${header[ce-partitionkey]}
+                steps:
+                  - set-header:
+                      name: kafka.PARTITION_KEY
+                      simple: ${header[ce-partitionkey]}
+        - to:
+            parameters:
+              configuration: '#bean:{{kafka-ssl-sink-configuration-local}}'
+            uri: '{{kafka-ssl-sink-local}}:{{topic}}'
```
[the remark this answers is not in the document]
Comment (oscerd) [reply kamelets/kafka-ssl-sink.kamelet.yaml:167]: Uri is part of the yaml DSL and of the model, so it's not a parameter to list.

**Review by oscerd — CHANGES_REQUESTED** [review]
Just some minor findings, but overall it looks good. Thanks a bunch.

**Comment by oscerd** [comment]
Fixes #724

File: kamelets/kafka-ssl-sink.kamelet.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (agiertli) [reply kamelets/kafka-ssl-sink.kamelet.yaml:49]: Fixed via: https://github.com/apache/camel-kamelets/pull/881/commits/1913f74240188a4bc2bd03df181acc9e01c2532f

File: kamelets/kafka-ssl-sink.kamelet.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (agiertli) [reply kamelets/kafka-ssl-sink.kamelet.yaml:98]: Fixed via: https://github.com/apache/camel-kamelets/pull/881/commits/e33cbcc87d61d39d0c6e1ecd94b48aaf2bd245e8

File: kamelets/kafka-ssl-sink.kamelet.yaml (hunk 3 above)
[the remark this answers is not in the document]
Comment (agiertli) [reply kamelets/kafka-ssl-sink.kamelet.yaml:174]: @oscerd if I understand correctly no change is needed from my side on this one?

[the remark this answers is not in the document]
Comment (oscerd) [reply kamelets/kafka-ssl-sink.kamelet.yaml:173]: This is fine

**Review by oscerd — APPROVED** [review]

**Pull request merged**